qml/pages/FirstPage.qml

```diff
@@ -169,7 +169,8 @@ Page {
                                           spamop: topic.posters[0].user_id,
                                           user_id: spammerop,
                                           has_accepted_answer: topic.has_accepted_answer,
-                                          highest_post_number: topic.highest_post_number
+                                          highest_post_number: topic.highest_post_number,
+                                          notification_level: (loggedin.value == "-1") ? "-1" : topic.notification_level

```

Comment:
I'm getting a ton of muted icons/statuses, probably needs: topic.notification_level !== undefined ? topic.notification_level : 1 to default to normal state